PYME/LMVis/Extras/pointSetGeneration.py

```diff
@@ -139,37 +170,83 @@ def refresh_choices(self):
-    
+
+    helpInfo = {
+        'source' : '''
+Select the type of point source to generate points.
+A wormlike source, an image based source and a file based source are supported.
+''',
+        'meanIntensity' : '''
+This parameter specifies the mean number of photons in an event.
+Typically values are in the range from 100 to several 10000.
+''',
+        'meanDuration' : '''
+The mean duration of events which is specified in units of frames.
+''',
+        'meanTime' : '''
+This parameter, the mean time of the series in frame units, is related to the series duration.
+For PAINT mode this is half the duration of the series, for STORM simulation mode the relationship is a little more complex.
+There it has something to do with the time where most of the events are occuring. 
+''',
+        'meanEventNumber' : '''
+This parameter specifies the mean number of times an event occurs at a single marker location.
+''',
+        'backgroundIntensity' : '''
+The background intensity in units of photons, typically in the range from a few tens to hundreds of photons.
+''',
+        'mode' : '''
+With the simulation mode you can choose between STORM or PAINT mode. 
+This parameter effects how event rate changes with time (it stays constant in PAINT mode).
+It has little effect otherwise.
+'''
+    }
+
+    def helpStr(self, name):
+        def cleanupHelpStr(str):
+            return str.strip().replace('\n', ' ').replace('\r', '')
+        
+        return cleanupHelpStr(self.helpInfo[name])
+        
     def default_traits_view( self ):
         from traitsui.api import View, Item, EnumEditor, InstanceEditor
+
         traits_view = View( Item( 'source',
                             label= 'Point source',
                             editor =
                             InstanceEditor(name = 'sources',
                                 editable = True),
+                            help=self.helpStr('source'),
                                 ),
                         Item('_'),
-                        Item('meanIntensity'),
-                        Item('meanDuration'),
-                        Item('meanEventNumber'),
-                        Item('meanTime'),
+                        Item('meanIntensity',tooltip='mean photon number of events',
+                             help=self.helpStr('meanIntensity')),
+                        Item('meanDuration',tooltip='mean duration of events in units of frames',
+                             help=self.helpStr('meanDuration')),
+                        Item('meanEventNumber',tooltip='mean number of times events occurs at a single marker location',
+                             help=self.helpStr('meanEventNumber')),
+                        Item('meanTime',tooltip='mean time of the series, roughly related to series duration, in frame units',
+                             help=self.helpStr('meanTime')),
+                        Item('scaleFactor',tooltip='unsure about this parameter'),
```

Comment:
Probably leave out of view ... this is the standard deviation of the PSF in pixels. It is used (in combination with the background intensity, which is per pixel) to derive a detection threshold.